table/get_context.h

```diff
@@ -72,6 +74,8 @@ class GetContext {
     return true;
   }
 
+  void record_counters(Tickers ticker, size_t val);
```

Comment:
the underscore-separated lowercase naming convention should only be for functions that simply return a value. For more complex ones, let's follow the capitalizing convention, like in `NeedToReadSequence()`.